### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by Xpl0itU**

### CODE REVIEW

**Comment by V10lator** [comment]
Why the Makefile changes?

**Comment by Xpl0itU** [comment]
libiosuhax is copied to wut/usr, I even have to make that change in my local machine for NUSspli to compile

**Comment by V10lator** [comment]
That's actually a problem of wiiu-env using a path not officialy used with WUT, see:
https://github.com/wiiu-env/libiosuhax/blob/master/Makefile#L114
https://github.com/devkitPro/wut/blob/master/samples/make/helloworld/Makefile#L62
So you should ask @Maschell why it's done this way and what's the plan to fix this upstream.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+RUN make -j$(nproc) install
+WORKDIR /curl/include
+RUN make -j$(nproc) install
+WORKDIR /
+
+# build libiosuhax
+FROM builder AS libiosuhaxbuild
+ENV WUT_ROOT=$DEVKITPRO/wut
+
+RUN git clone --recursive https://github.com/wiiu-env/libiosuhax
```
Comment (V10lator) [inline Dockerfile:177]: Please change the URL to https://github.com/Crementif/libiosuhax

**Comment by eku** [comment]
Am I wrong, or could the number of layers be drastically reduced by combining all RUN commands?

**Pull request merged**